Automattic/wp-calypso
# Store: Add the ability to edit and delete review replies

This PR adds the ability to delete or edit an admin reply.

<img width="778" alt="screen shot 2017-09-26 at 5 18 42 pm" src="https://user-images.githubusercontent.com/689165/30889978-c7c18158-a2de-11e7-8bd0-34c828ff1b06.png">


To Test:
* Load up this branch. The product review API changes have been merged and deployed to AT servers, so you don't need a special version of `wc-api-dev`.
* Run `npm run test-client client/extensions/woocommerce/state` and make sure all tests pass.
* Make sure you have a review with a couple of replies added. You can add these in `wp-admin` right now under the product edit page.
* Go to `http://calypso.localhost:3000/store/reviews/:site` and find your review with replies.
* Expand the review.
* Click `Edit Reply` and the display should update to a textarea.
* Make some changes, and hit save. Make sure you see a success message and the contents update.
* Click `Delete Reply` and click OK on the approval dialog.
* Make sure you see a success message and the reply is removed.
